backend/db/db.go

```diff
@@ -63,6 +63,8 @@ func (c *Database) CreateNewArticle(url string, title string, source string) (st
 }
 
 // GetAllArticles gets all articles from db, formatted to Article type
-func (c *Database) GetAllArticles() ([]Article, error) {
-	return c.FindAllArticles()
+// numArticles: number of Articles to return from Collection,
+// NOTE:  If -1 is passed in, we return ALL Articles
+func (c *Database) GetAllArticles(numArticles int) ([]Article, error) {
```

Comment:
Hmm can we define a seperate function for this?

The name `GetAllArticles` suggests we are getting all articles. Lets define a separate function that takes in a parameter. We can do some refactoring in `mongo.go` to help us with the common code

This way we don't have to carry a special behavior for the value of `-1`